BlazorGrpcWebCodeFirst/Server/Services/WheatherService.cs

```diff
@@ -0,0 +1,37 @@
+using ProtoBuf.Grpc;
+using System.Threading.Tasks;
+using BlazorGrpcWebCodeFirst.Shared;
+using System;
+using System.Linq;
+
+namespace BlazorGrpcWebCodeFirst.Server.Services;
+
+#region snippet
+public class WheatherService : IWeatherForecasts
```

Comment:
Typo in service name, should be `WeatherService` without `h`.

Reply:
Oh! thanks about review.